core/src/main/java/org/apache/iceberg/AllManifestsTable.java

```diff
@@ -44,13 +44,17 @@
  */
 public class AllManifestsTable extends BaseMetadataTable {
   private static final Schema MANIFEST_FILE_SCHEMA = new Schema(
+      Types.NestedField.required(14, "content", Types.IntegerType.get()),
       Types.NestedField.required(1, "path", Types.StringType.get()),
       Types.NestedField.required(2, "length", Types.LongType.get()),
       Types.NestedField.optional(3, "partition_spec_id", Types.IntegerType.get()),
       Types.NestedField.optional(4, "added_snapshot_id", Types.LongType.get()),
       Types.NestedField.optional(5, "added_data_files_count", Types.IntegerType.get()),
       Types.NestedField.optional(6, "existing_data_files_count", Types.IntegerType.get()),
       Types.NestedField.optional(7, "deleted_data_files_count", Types.IntegerType.get()),
+      Types.NestedField.required(15, "added_delete_files_count", Types.IntegerType.get()),
```

Comment:
If we did this initially, we'd probably just have one set of counts with a generic name but this is public and widely used.